lemur/certificates/views.py

```diff
@@ -542,6 +543,7 @@ def __init__(self):
         super(CertificatesUpload, self).__init__()
 
     @validate_schema(certificate_upload_input_schema, certificate_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
this restriction doesn't make much sense to me. I don't understand the risk of a user uploading a cert they already own

Reply:
we are currently not enforcing the default `read-only` role. a user assigned the `read-only` role is still able to upload certificates. we are thinking if an admin creates a new user with the `read-only` role. they shouldn't be able to create/upload any certificates just read them, unless they are promoted to the `operator` or `admin` role.

also, if a user was initially assigned the `operator` role, then changed to the `read-only` role. this would ensure the user would not be able to create/upload any certificates.